tasks/init_cfme.yml

```diff
@@ -0,0 +1,33 @@
+- name: Add host alias of appliance
+  no_log: true
+  add_host:
+    hostname: "{{ miq_ip_addr }}"
+    ansible_host: "{{ miq_ip_addr }}"
+    ansible_password: "{{ miq_password }}"
+    ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+
+- name: Fetch info about appliance
+  command: "rpm -qi cfme"
+  register: cfme_rpm
+  ignore_errors: yes
+  warn: no
+  delegate_to: "{{ miq_ip_addr }}"
+
+- block:
+
+  - name: Hotplug database disk for CFME
+    ovirt_disk:
+      auth: "{{ ovirt_auth }}"
+      name: "{{ item.name }}"
+      vm_name: "{{ miq_vm_name }}"
+      interface: "{{ item.interface | default(omit) }}"
+      size: "{{ item.size | default(omit) }}"
+      format: "{{ item.format | default(omit) }}"
+      storage_domain: "{{ item.storage | default(miq_vm_disk_storage) | default(disk_storage_domain.name) }}"
+    with_items: "{{ miq_vm_disks }}"
+  
+  - name: Initialize CloudForms
+    command: "appliance_console_cli -i -r 0 -U root -p {{ miq_password }} -b /dev/vdb -k -f"
```

Comment:
can we also pass the command line as parameter, so we can actually give the user the option to manipulate the installation procedure to be more modular.